roles/php/tasks/mod-php.yml

```diff
@@ -4,3 +4,9 @@
     dest: "/etc/php/{{ php_version }}/apache2/php.ini"
     regexp: "date.timezone ="
     line: "date.timezone = {{ server.timezone }}"
+
+- name: php | ensure memory limit is set in apache2 php.ini
+  ansible.builtin.lineinfile:
+    dest: "/etc/php/{{ php_version }}/apache2/php.ini"
+    regexp: "memory_limit ="
+    line: "memory_limit = {{ memory_limit }}"
```

Comment:
Per discussion in Ansible Open House, maybe use `php_memory_limit` as the var, so the association is clear.